back/engines/commercial/id_vienna_saml/app/lib/id_vienna_saml/citizen_saml_omniauth.rb

```diff
@@ -34,6 +36,10 @@ def profile_to_user_attrs(auth)
       }
     end
 
+    def profile_to_uid(auth)
+      auth.dig(:extra, :raw_info).to_h[USERID_KEY]&.first
```

Comment:
Is it expected to return `nil` in some cases? What happens then?

Reply:
I don't think it can be `nil`, I added it just in case, similarly how we do it for `first_name`, `last_name` above. But maybe indeed it just becomes more confusing, I'll remove `&`.